### ISSUE TITLE
nemo rename multiple files faster

### ISSUE THREAD

**Issue by Hug0122** [linuxmint/nemo#3136]
Hello,
I have been using Linux for many years on a personal basis and windows at work. A feature that I use on Windows I miss a lot when I am on Linux (whatever the flavor).
In the Windows file browser, when I want to rename several files (be careful I'm not talking about mass nenaming), I click on the file, I type on the "F2" key to enter edition mode, and I rename my file . But instead of typing the "Enter" key to validate, I type on the "Tab" key, and this goes on the next file where I can directly modify the name. In fact we stay in "edition" mode as long as we did not hit the "Enter" key.
It is not a feature that revolutionizes the world, but I assure you that when I return in the evening on Linux at home, I miss the function very much. It's just "simpler".
I do not know what this represents in development, but it is a feature that I would one day see on Linux. (To my knowledge, no flavor offers this functionality on Linux.)
Cordially

**Comment by clefebvre** [comment]
It's an interesting idea but hard to discover and pretty niche in my opinion. We also provide a useful batch renamer program called bulky which makes renaming easy.

That said it's probably pretty easy to do. Pull requests are welcome if someone wants to do it.

**Comment by Hug0122** [comment]
Hello

Precisely, the purpose of this feature is not mass renaming. 
Imagine a directory containing 10 photos from a camera (photo001, photo002,... photo010).
You want to give a name to each of these pictures : eiffel tower, arc de triomphe, tour montparnasse,...
Mass renaming does not work here.
Here are the steps I do on linux : 
1] click on the first picture;
2] press "F2" key ;
3] rename the picture;
4] "Enter" key;
5] "down arrow" key or click to select the second photo;
2] "F2" key
3]...
4]...
5]...

Here are the steps I do on Windows :
1] click on the first picture;
2] "F2" key or second click on the same file to enter "edit" mode;
3] rename the photo;
4] tabulation to go to the next photo;
3] rename the second photo;
4]...
3]...
...
5] "Enter" key or click in the blank of the page to exit the "edit" mode.

We save 2 steps out of 4 on each renaming (except the first file)
After 10 photos or more, I guarantee that you will appreciate the feature.

Moreover I mentioned above another facility that I find appreciable. It is the opening of the editing mode via two clicks: one to select and one to enter the editing mode. Be careful, it is not the traditional double-click to open the file. There has to be a very small time lapse between the two clicks to distinguish the two modes.

`It's an interesting idea but hard to discover and pretty niche in my opinion.`
Yes of course, I agree with you, these are features that are not necessarily visible. If you don't know them, you don't use them.
Beyond the practical side of these features, I think that if we want to attract more Windows users (I mean basic users, lambda), we should not change their habits (man hates change), be inspired by the features that people are used to, the little things that make life easier,... 
Moreover, they do not deprive themselves of this. Since Windows 10, Microsoft has set up its Windows Store, which is very similar to our application store. We wonder where they got this idea from. :)
It's because the Cinnamon environment looks a lot like Windows 7 and is "naive user" oriented, that I was able to install Linux Mint on my parents' computer (but it wasn't easy, I had to tell them that W7 had become obsolete and that there could be security flaws to make them change their mind).

Regards
